assets/js/modules/reader-revenue-manager/components/dashboard/RRMSetupSuccessSubtleNotification.js

```diff
@@ -0,0 +1,141 @@
+import {
+	MODULES_READER_REVENUE_MANAGER,
+	PUBLICATION_ONBOARDING_STATES,
+} from '../../datastore/constants';
+
+function RRMSetupSuccessSubtleNotification() {
+	const [ notification, setNotification ] = useQueryArg( 'notification' );
+	const [ slug, setSlug ] = useQueryArg( 'slug' );
+
+	const publicationOnboardingState = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getPublicationOnboardingState()
+	);
+
+	const publicationID = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getPublicationID()
+	);
+
+	const serviceURL = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getServiceURL( {
+			publicationID,
+		} )
+	);
+
+	const onDismiss = () => {
+		setNotification( undefined );
+		setSlug( undefined );
+	};
+
+	if (
+		'authentication_success' !== notification ||
+		slug !== 'reader-revenue-manager' ||
+		publicationOnboardingState === undefined
+	) {
+		return null;
+	}
+
+	if (
+		publicationOnboardingState ===
+		PUBLICATION_ONBOARDING_STATES.ONBOARDING_COMPLETE
+	) {
+		return (
+			<SubtleNotification
+				title={ __(
+					'Your Reader Revenue Manager account was successfully set up!',
+					'google-site-kit'
+				) }
+				description={ __(
+					'Unlock your full reader opportunity by enabling features like subscriptions, contributions and newsletter sign ups in the Reader Revenue Manager settings.',
+					'google-site-kit'
+				) }
+				onDismiss={ onDismiss }
+				dismissLabel={ __( 'Maybe later', 'google-site-kit' ) }
+				ctaLink={ serviceURL }
+				ctaLabel={ __( 'Customize settings', 'google-site-kit' ) }
+				isCTALinkExternal
+			/>
+		);
+	}
+
+	if (
+		publicationOnboardingState ===
+		PUBLICATION_ONBOARDING_STATES.PENDING_VERIFICATION
+	) {
+		return (
+			<SubtleNotification
+				title={ __(
+					'Your Reader Revenue Manager account was successfully set up!',
+					'google-site-kit'
+				) }
+				description={ __(
+					'Your publication is still awaiting review, you can check its status in Reader Revenue Manager.',
+					'google-site-kit'
+				) }
+				onDismiss={ onDismiss }
+				dismissLabel={ __( 'Got it', 'google-site-kit' ) }
+				ctaLink={ serviceURL }
+				ctaLabel={ __( 'Check publication status', 'google-site-kit' ) }
+				isCTALinkExternal
+			/>
+		);
+	}
+
+	if (
+		publicationOnboardingState ===
+		PUBLICATION_ONBOARDING_STATES.ONBOARDING_ACTION_REQUIRED
+	) {
+		return (
+			<SubtleNotification
+				title={ __(
+					'Your Reader Revenue Manager account was successfully set up, but your publication still requires further setup in Reader Revenue Manager.',
+					'google-site-kit'
+				) }
+				onDismiss={ onDismiss }
+				dismissLabel={ __( 'Got it', 'google-site-kit' ) }
+				ctaLink={ serviceURL }
+				ctaLabel={ __(
+					'Complete publication setup',
+					'google-site-kit'
+				) }
+				isCTALinkExternal
+				variant="warning"
+			/>
+		);
+	}
+
+	return null;
+}
+
+export default whenActive( { moduleName: 'reader-revenue-manager' } )(
```

Comment:
I will do it myself to avoid another round.
```suggestion
export default whenActive( { moduleName: READER_REVENUE_MANAGER_MODULE_SLUG } )(
```